### PR TITLE
add EKS Anywhere to products in ecosystem

### PR DESCRIPTION
**Pull request by makkes**

### CODE REVIEW

File: content/en/ecosystem.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ We are happy and proud to have you all as part of our community! :sparkling_hear
 
 | Vendor      | Product / Service       | Link                                                               |
 |-------------|-------------------------|--------------------------------------------------------------------|
+| AWS         | EKS Anywhere            | [Documentation](https://anywhere.eks.amazonaws.com/docs/)          |
```
Comment (hiddeco) [inline content/en/ecosystem.md:21]:
```suggestion
| AWS         | EKS Anywhere            | [Documentation](https://anywhere.eks.amazonaws.com/docs/tasks/cluster/cluster-flux//)          |
```

Comment (makkes) [reply content/en/ecosystem.md:21]: Had the same thought but then decided to align the link with the other ones that link to the top-level docs but I'm happy to deep-link here.

**Comment by makkes** [comment]
Dang, the mobile app didn't let me edit the commit msg. I'll squash the commits in a sec

**Review by dholbach — APPROVED** [review]
Thanks for the PR!

**Pull request merged**